Update Governance and Contributing doc (#3198)
OpenRefine/OpenRefine: GOVERNANCE.md

- Informing developers of strengths and weaknesses from a new user perspective through the [user discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)
- Providing moral support (a ‘thank you’ goes a long way)
- Writing tutorials
- Make financial contribution to OpenRefine //TO DO add link 

How to become an OpenRefine user? [Download OpenRefine](http://openrefine.org/download.html) and start refining!

### Guest Contributors

> Guest Contributors are users contributing in concrete ways to the project. They are one-time participation or are new contributors to the project. Contribution can take place in many ways:

Author: New suggestion: 
> Guest Contributors are new users getting familiar with the projects or users that have not expressed they interest to be involved with the Organization.

> Guest Contributors are users contributing in concrete ways to the project. They are one-time participation or are new contributors to the project. Contribution can take place in many ways:

Reviewer: I don't know if I would even put anything about Contributors in the GOVERNANCE.md doc.  I think maybe this section can go into the CONTRIBUTING.md doc with my further reply notes about the decision process, etc.

Here in the GOVERNANCE.md doc, I think we can just mention that the Advisory or Steering committee have the responsibility to vote to allow someone to be added to the GitHub Admin team or Core Contributors team.  Something simple 1 line about that vote responsibility.  And the rest of Roles, Access, GitHub, Contributing, can all go into CONTRIBUTING.md  I.E. stick to only Governance matters in the GOVERNANCE.md doc.

> Guest Contributors are users contributing in concrete ways to the project. They are one-time participation or are new contributors to the project. Contribution can take place in many ways:

Author: The goal of the governance document is to present the different roles and responsibilities in the project. It is the right place to explain the difference between a Guest Contributor (with no access to manage the repo) and a Contributor (who can manage the repo)

- Supporting new users via the [user discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine)
- Submitting patches to fix bugs or add features
- Identifying requirements
- Providing graphics and web design
- Writing and maintaining the [documentation](https://github.com/OpenRefine/OpenRefine/tree/master/docs)

Guest Contributors abide by the project’s [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md)

How to become an OpenRefine contributors? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)

### Organization Member - Contributors 
If you make regular contribution to OpenRefine, you will most likely become a Organization Member - Contributor.

> Organization member have no more authorities than contributors, and they should engage with the community through the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open) regarding their intention. However, Organization Member have earned enough trust from the community to have direct access to the project code base without having to submit pull request. Organization Member seek approval after the contribution is made, rather than before.

Reviewer 2: Yes, agreed. Please update to match wetneb's comment about the current practice

> Organization member have no more authorities than contributors, and they should engage with the community through the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open) regarding their intention. However, Organization Member have earned enough trust from the community to have direct access to the project code base without having to submit pull request. Organization Member seek approval after the contribution is made, rather than before.

Reviewer 3: This is still not clear enough but I'll open another PR to get this right.

Therefore Organization Member:

- Help contributors via the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev).
- Manage tag, rename and organize issues, pull requests and [projects](https://github.com/OpenRefine/OpenRefine/projects)
- Have direct access to the code base to create new branches
- Organize the wiki
- Merge pull requests submitted by contributors

Organization Member Contributors abide by the project’s [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md)

#### How to be part of the OpenRefine Organization Member?  
Be a guest contributor and be nominated as a Organization Member. Current Organization Member selects and elects new Organization Member. You may nominate yourself. Nomination should be sent to the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev)

#### Current List of Organization Member 
The list is available here: https://github.com/orgs/OpenRefine/people. 

### Release Manager
The Release Manager is responsible to 
* Coordinate with the community to select which issue and pull request are part of the release

> * Prepare and publish the release 

Reviewer: They might not do it themselves but use a backup or designated peer, like the current situation with Tom and Antonin, and we should not have wording that does not allow backup managers. So I suggest a more open rephrasing of:
```suggestion
* Prepare and coordinate publishing the release 
```

* Act as an advocate for the Project in events and support the project’s communication online, as time allows.

#### How to be part of the Steering Committee
Steering Committee are invited by the Advisory Committee

#### Current List of Steering Committee Members
* Chris Erdmann (University of North Carolina, RENCI)
* Alicia Fagerving (Wikimedia Sverige)
* Sandra Fauconnier (Wikimedia Foundation)
* Rufus Pollock (Datopian, Open Knowledge Foundation)
* Simon Rogers (Google News Initiative)
* Lozana Rossenova (Rhizome)
* Juliane Schneider (Harvard Catalyst, Clinical and Translational Science Center)
* Wesley Sherperd (Unifyd Insights)
* Fabio Tacchelli (Siren Solutions)
* Ruben Verborgh (University of Ghent)

### Advisory Committee
The Advisory Committee runs the adminstrative aspect of the project on a day to day basis with the support of Code for Science and Society (CS&S). Its member are bound by the  fiscal sponsorship agreement with Code for Science and Society. Meet once per month with Code for Science and Society. 

> It is composed of at least three members. No more than 49 percent of the members of the Advisory Committee may be paid by the projcet.

Reviewer: ```suggestion
It is composed of at least three members. No more than 49 percent of the members of the Advisory Committee may be paid by the project.
```

Advisory Committee members: 
* Represent OpenRefine with Code for Science and Society 
* Provide guidance and oversight of the Project’s staff and operations;
* Approve budgets, and contracts
* Vote to terminate contracts when necessary.

> * Participate to at least one call per quarter;

Reviewer: ```suggestion
* Participate in at least one call per quarter;
```

* Help advocate for the Project;
* Help the Project build connections and partnerships;

> * Are part of the Admin team for the project on Github

Reviewer: Hmm, I don't think there needs to be an explicit requirement that advisory committee members MUST be part of the Admin team.  But if others disagree, that's fine. We just need to get consensus on what qualifies to be an Admin for the GitHub project or not.  Perhaps Roles and Responsibilities on GitHub can be put into a new section.  I don't know.

```suggestion
* Can be part of the Admin team for the project on GitHub
```

Author: Good point @thadguidry  - In that case I will remove the mention of the GitHub admin team entirely.

### Manage Funds 

While the vast majority of OpenRefine contributors are volunteers, we recognized that need to attract and retain contributors to either
- perform time critical maintenance tasks for the project (for example release management, security update, pull request review and comments)
- address issue that will not be naturally done by the communittee (for example documentation or on long terms project like OpenRefine 4.x)

CS&S currently manage the current funding sources: 
- **Grants**: Funds are allocated based on the grants requierements. Grants help to secure resources to achieve long term goal or supports software maintenance, growth, development, and community engagement.
- **Google Summer of Code**: Google Summer of Code mentors are eligible to a USD 500 compensation for their work. 
- **Donations made through Github sponsors or done directly to CS&S on behalf of OpenRefine**: Usage of the fund is made by the Advisory committee with guidance from Organization Members. 

> The CS&S, the Steering and Advisory comittees have experience applying and managing grants. If you are interested in applying to a grant to improve OpenRefine, please share your idea early on the Developer mailing list. It will be a pleasure helping you through your grant application. New grants must be approved by the Advisory Committee. that CS&S retains a 15% handling fees on any contribution to finance their operations (20% for government funding). 

Reviewer: Fix grammar.
Isn't contribution really to be worded as **donation?**
```suggestion
Combined with CS&S, the Steering and Advisory committees have experience applying and managing grants. If you are interested in applying to a grant to improve OpenRefine, please share your idea early on the Developer mailing list. It will be a pleasure helping you through your grant application. New grants must be approved by the Advisory Committee. that CS&S retains a 15% handling fees on any donation to finance their operations (20% for government funding). 
```

### Other assets 

> CS&S owns and manage the domain openrefine.org 

Reviewer: ```suggestion
CS&S owns and manages the domain openrefine.org 
```

- Providing moral support (a ‘thank you’ goes a long way)
- Writing tutorials

How to become an OpenRefine user? [Download OpenRefine](http://openrefine.org/download.html) and start refining!

### Contributors
Contributors are users contributing in concrete ways to the project. They are one-time participation or are new contributors to the project. Contribution can take place in many ways:

- Supporting new users via the [user discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine)
- Submitting patches to fix bugs or add features via pull request
- Identifying requirements
- Providing graphics and web design
- Writing and maintaining the [documentation](https://github.com/OpenRefine/OpenRefine/tree/master/docs)

Contributors abide by the project’s [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md)

How to become an OpenRefine contributor? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)

> ### Comitters 

Reviewer: 2 m's and 2 t's.  So change to `Committers` throughout the doc.

How to become an OpenRefine user? [Download OpenRefine](http://openrefine.org/download.html) and start refining!

### Contributors
Contributors are users contributing in concrete ways to the project. They are one-time participation or are new contributors to the project. Contribution can take place in many ways:

- Supporting new users via the [user discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine)
- Submitting patches to fix bugs or add features via pull request
- Identifying requirements
- Providing graphics and web design
- Writing and maintaining the [documentation](https://github.com/OpenRefine/OpenRefine/tree/master/docs)

Contributors abide by the project’s [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md)

How to become an OpenRefine contributor? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)

### Comitters 

> If you make regular contribution to OpenRefine, you will most likely become a comitter. 

Reviewer: However, I see this mention of "committer" as unnecessary.  I think we are using the wrong term here.
Commit is a `git` thing.
Contribution is a Project health metric we want to entertain and invite much more.

> Comitters have no more authorities than contributors, and they should engage with the community through the [developer discussion list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open) regarding their intention. However, Comitters have earned enough trust from the community to have direct access to the project code base without having to submit pull request. Comitter seek approval after the contribution is made, rather than before.

Reviewer: We should use the terms **Project Admins**, **Project Owners** and **Contributors**.
But I caution on even adding Project Owners into this Governance because we want Contributors to actually feel like Owners, being responsible for the health of the project and taking an active role and continuing in its development.  So, to that end, I really think we can simplify with only 2 levels...**Project Admins**, **Contributors** ?

Author: From @tfmorris  comment #3198#issuecomment-721289810, and in today's call I understood we agreed to contributors & committers.   I really like to close this PR and move to the next steps. 
If we can't get an agreement on the terminology by next Monday I ask for a vote on the dev mailing list (using a survey).

Reviewer: I am OK with the current terminology.
